dashboards-observability/.cypress/integration/event_analytics.spec.js

```diff
@@ -394,7 +394,7 @@ describe('Switch on and off livetail', () => {
     cy.get('[data-test-subj="searchAutocompleteTextArea"]').type(TEST_QUERIES[1].query);
 
     cy.get('[data-test-subj=eventLiveTail]').click();
-    cy.get('[data-test-subj=eventLiveTail__delay10]').click();
+    cy.get('[data-test-subj=10s]').click();
```

Comment:
any ways to make the test id naming more specific?

Reply:
+1 - if we can't find an intuitive ID to  call this by, could we extract this line to a function that describes the action? e.g `clickLivetailPlayButton()`

Reply:
i'll try to make it more specific, if not i'll try the function method too